src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs

```diff
@@ -23,32 +23,26 @@ public class NodeController : Controller, IUpdateModel
         private readonly IEnumerable<IAdminNodeProviderFactory> _factories;
         private readonly IAdminMenuService _adminMenuService;
         private readonly INotifier _notifier;
-
+        private readonly IHtmlLocalizer H;
         public NodeController(
             IAuthorizationService authorizationService,
             IDisplayManager<MenuItem> displayManager,
             IEnumerable<IAdminNodeProviderFactory> factories,
             IAdminMenuService adminMenuService,
             IShapeFactory shapeFactory,
-            IStringLocalizer<NodeController> stringLocalizer,
             IHtmlLocalizer<NodeController> htmlLocalizer,
             INotifier notifier)
         {
             _displayManager = displayManager;
             _factories = factories;
             _adminMenuService = adminMenuService;
             _authorizationService = authorizationService;
-
             New = shapeFactory;
             _notifier = notifier;
-            T = stringLocalizer;
             H = htmlLocalizer;
         }
 
         public dynamic New { get; set; }
```

Comment:
We can do the same for this property

Reply:
Ok, I saw you did it as getter property, no worry I will handle them too